reports/models.py

```diff
@@ -0,0 +1,402 @@
+        null=True,
+        db_column="occurrence_id",
+    )
+    occurrence_place_id = models.CharField(
+        max_length=255, verbose_name=_("place id"), blank=True
+    )
+    # The position of the (event) place from LinkedEvents API
+    occurrence_place_position = ArrayField(
+        models.DecimalField(max_digits=9, decimal_places=6),
+        size=2,
+        verbose_name=_("coordinates of the event occurrence place"),
+        null=True,
+    )
+    # The divisions of the (event) place from LinkedEvents API
+    occurrence_place_divisions = ArrayField(
+        models.CharField(max_length=200),
+        verbose_name=_("divisions of the occurrence place"),
+        help_text=_("Open Civic Data identifiers"),
+        null=True,
+    )
+    occurrence_languages = ArrayField(
+        # ArrayField Size 2: Language id, translated name
+        ArrayField(models.CharField(max_length=255), size=2),
+        verbose_name=_("languages of the occurrence"),
+    )
+    occurrence_cancelled = models.BooleanField(
+        verbose_name=_("cancelled status of the occurrence")
+    )
+    occurrence_amount_of_seats = models.PositiveSmallIntegerField(
+        verbose_name=_("amount of the seats in the occurrence")
+    )
+
+    occurrence_start_time = models.DateTimeField(
+        verbose_name=_("occurrence start time")
+    )
+    occurrence_end_time = models.DateTimeField(verbose_name=_("occurrence end time"))
+
+    # PalvelutarjotinEvent
+    # - If the foreign key integrity is gone,
+    # keep some information so the data is still available for reports:
+    #     1. Enrolment start time
+    #     2. LinkedEvent id
+    # - Store the status of the type of the enrolment
+    linked_event_id = models.CharField(
+        max_length=255, verbose_name=_("linked event id")
+    )
+    enrolment_start_time = models.DateTimeField(
+        verbose_name=_("enrolment start time"), null=True
+    )
+    # enrolment_externally tells if the
+    # PalvelutarjotinEvent.external_enrolment_url is filled
+    enrolment_externally = models.BooleanField(
+        verbose_name=_("external enrolments"), default=False
+    )
+
+    """
+    LinkedEvents Event information
+    """
+    provider = models.CharField(verbose_name=_("provider"), max_length=512, null=True)
+    publisher = models.CharField(
+        max_length=255,
+        help_text=_("a primary name of the publisher, e.g. a legally recognized name"),
+        null=True,
+    )
+    keywords = ArrayField(
+        # ArrayField Size 2: id, translated name
+        ArrayField(models.CharField(max_length=255), size=2),
+        verbose_name=_("keywords"),
+        null=True,
+    )
+
+    # Distance
+    distance_from_unit_to_event_place = models.DecimalField(
+        max_digits=9,
+        decimal_places=2,
+        verbose_name=_("distance from unit to event place"),
+        null=True,
+    )
+
+    objects = EnrolmentReportManager()
+
+    @property
+    def study_group_group_size(self):
```

Comment:
Is this `study_group_group_size` property used somewhere? Should it also include the amount of adults?

Reply:
The real name of the children count field in study_group model is "group_size". That's a missleading name, so I wanted to rename it to the enrolment report model. I added this getter there for consistensy reasons.

Reply:
so, group_size is not children + adults - it's just the children. Bad name from history.